src/Components/App/App.js

```diff
@@ -116,32 +118,27 @@ const App = () => {
   }, [])
 
   return(
-    <Routes>
-      <Route path="/" element={
+    <div>
+      <Header togView={toggleView} currentUser={currentUser}/>
+      <Routes>
+        <Route path="/" element={
           <div>
-            <Header togView={toggleView} currentUser={currentUser}/>
             <Search vendors={vendors} search={searchResults} reset={resestResults} allSearch={searchButtons}/>
-            <Results vendors={vendors} remFav={removeFav} addFav={favTruck} favorites={favorites} dist={importDistance}/>
-          </div>
-      } />
-      <Route path="/vendor/:id" element={
-          <div>
-            <Header togView={toggleView} />
-            {/* <TruckDetails /> */}
-            {/* //params.match, need to pass down the id that is clicked as a prop to truck details to element specfic truck info */}
+            <Results vendors={vendors} remFav={removeFav} addFav={favTruck} favorites={favorites} />
           </div>
-      } />
+        } />
+      <Route path="/vendor/:id" element={<TruckDetails vendors={vendors}/>}
+      />
       <Route path="/vendor-view" element={
-          <div>
-            <Header togView={toggleView} />
-            <VendorView toggleLive={sendMessage} vendor={vendors.find(v => v.id == 1)} />
-          </div>
+        <div>
+          <VendorView toggleLive={sendMessage} vendor={vendors.find(v => v.id == 1)} currentVendor={currentVendor}/>
```

Comment:
regarding the vendor view truck details we might just be able to just hard code and pass through one vendor and just stylize it based upon that rather than render in truck details for this, my mind immediatly goes to utilizing components whereever we can but i think for the sake of this it might be easiest to just hardcode and pass one down and the nstylize it based upon that